src/pages/community/index.mdx

```diff
@@ -25,8 +25,7 @@ Also check out:
 * **User mailing list:** [druid-user@googlegroups.com](https://groups.google.com/forum/#!forum/druid-user) for general
 discussion, questions, and announcements.
 * **LinkedIn** Connect with other Apache Druid professionals in the [LinkedIn group](https://www.linkedin.com/groups/8791983/).
-* **Meetups:** Check out [Apache Druid on meetup.com](https://www.meetup.com/topics/apache-druid/) for links to regular
-meetups in cities all over the world.
+* **Meetups:** Druid Meetups can be found all over the world, to find out more about Druid Meetups and to find your local one see [Druid Meetups](/community/meetups).
```

Comment:
Will this end up in the nav? Or will it just be a separate page?

Reply:
Separate page